bigframes/session/__init__.py

```diff
@@ -749,6 +750,34 @@ def read_gbq_table(
             filters=filters,
         )
 
+    def read_gbq_table_streaming(
+        self, table: str
+    ) -> bigframes.streaming.StreamingDataFrame:
+        """Turn a BigQuery table into a StreamingDataFrame.
+
+        Note: The bigframes.streaming module is a preview feature, and subject to change.
+
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            >>> sdf = bpd.read_gbq_table_streaming("bigquery-public-data.ml_datasets.penguins")
+        """
+        warnings.warn(
+            "The bigframes.streaming module is a preview feature, and subject to change.",
+            stacklevel=1,
+            category=bigframes.exceptions.PreviewWarning,
+        )
+
+        from bigframes import streaming
+
+        df = self._read_gbq_table(
+            table, api_name="read_gbq_table_steaming", enable_snapshot=False
+        )
```

Comment:
I think this will probably setup a default index? Should probably explicitly set a null index to ensure don't end up putting a window function into the final sql. Also, do we need to validate that the table is streaming-compatible? I'm guessing some table types don't work

Reply:
Done for NULL index. Easier SQL generated. Thanks.

For table types, it should be fine to let BQ emit errors. Do you have an idea how we can validate those? I don't see a clear way.

Reply:
we fetch metadata for each table when we read into bigframes. My guess is only "real" bq tables work with streaming, and probably not views, external tables, etc.